docs/conf.py

```diff
@@ -0,0 +1,146 @@
+# This code is part of a Qiskit project.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=invalid-name
+
+"""Sphinx documentation builder."""
+
+# -- General configuration ---------------------------------------------------
+import datetime
+import doctest
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+sys.path.append(os.path.abspath("."))
+
+project = "Qiskit Algorithms"
+copyright = f"2017-{datetime.date.today().year}, Qiskit Algorithms Development Team" # pylint: disable=redefined-builtin
```

Comment:
Hm, should this be only 2023 for now?

```suggestion
project_copyright = f"{datetime.date.today().year}, Qiskit Algorithms Development Team"
```

Reply:
This code/docs is all copyright back to then so 2017-2023 is correct.